src/components/Dao/Wiki/DaoWiki.tsx

```diff
@@ -0,0 +1,203 @@
+        props.showNotification(NotificationStatus.Failure, "You must be logged in to use Wiki!");
+        return;
+      }
+      const web3Provider = await getWeb3Provider();
+      const { dao, address, schemeParams, id } = states.find(hasWikiScheme);
+      setWikiSchemeAddress(id);
+      const { contractToCall } = schemeParams as IGenericSchemeParams;
+      const daoInformation: IDaoInformation = {
+        dao,
+        scheme: address,
+        contractToCall,
+      };
+      const dispatcher = new CustomDispatcher(wikiMethods, daoInformation);
+      renderWikiComponent(web3Provider, dispatcher);
+    }
+  };
+
+  React.useEffect(() => {
+    checkIfWikiSchemeExists();
+  }, []);
+
+  const registerWikiScheme = async () => {
+    if (!(await enableWalletProvider({ showNotification: props.showNotification }))) {
+      return;
+    }
+
+    const checkProposals = (proposal: Proposal) => {
+      const state = proposal.staticState as IProposalState;
+      return state.title === "Creation of WikiUpdate scheme";
+    };
+
+    const wikiProposalAlreadyExists = proposals.some(checkProposals);
+    const dao = props.daoState.address;
+    const schemeRegistrar = schemes.find((scheme: Scheme) => scheme.staticState.name === "SchemeRegistrar");
+
+    if (wikiProposalAlreadyExists) {
+      props.showNotification(NotificationStatus.Success, "Wiki Scheme proposal has already been created!");
+      props.history.replace(`/dao/${dao}/scheme/${schemeRegistrar.id}`);
+    } else {
+      const proposalValues = {
+        dao,
+        type: IProposalType.SchemeRegistrarAdd,
+        permissions: "0x" + (17).toString(16).padStart(8, "0"),
+        value: 0,
+        tags: ["Wiki"],
+        title: "Creation of WikiUpdate scheme",
+        description: "This will allow DAO to have Wiki functionality",
+        parametersHash: "0x00000000000000000000000000000000000000000",
+        scheme: schemeRegistrar.staticState.address,
+        schemeToRegister: "0xc8e06c1b6fb9a60f727c538233dac6ff5c1ddbf8", // rinkeby
+      };
+      await createProposal(proposalValues);
+    }
+  };
+
+  const NoWikiScheme = (
+    <div className={proposalStyle.noDecisions}>
+      <img className={proposalStyle.relax} src="/assets/images/yogaman.svg" />
+      <div className={proposalStyle.proposalsHeader}>Wiki scheme not registered on this DAO yet</div>
+      <p>You can create the proposal to register it today! (:</p>
+      <div className={proposalStyle.cta}>
+        <Link to={"/dao/" + props.daoState.address}>
+          <img className={proposalStyle.relax} src="/assets/images/lt.svg" /> Back to home
+        </Link>
+        <a
+          className={classNames({
+            [proposalStyle.blueButton]: true,
+          })}
+          onClick={registerWikiScheme}
+          data-test-id="createProposal"
+        >
+          + Register wiki scheme
+        </a>
+      </div>
+    </div>
+  );
+
+  return (
+    <div>
+      <Sticky enabled top={50} innerZ={10000}>
```

Comment:
Is it possible to do this without using `Sticky`?  We are wanting to ditch that component.  CSS should behave more cleanly anyway.

Reply:
Yes, 100%. Did it because I saw it in another component before, will remove it